library/src/main/java/com/mapzen/valhalla/TransitInfo.kt

```diff
@@ -54,8 +54,12 @@ class TransitInfo {
     return json.optString(KEY_SHORT_NAME)
   }
 
-  fun getColor(): Int {
-    return json.getInt(KEY_COLOR)
+  fun getColor(): Int? {
+    if (json.has(KEY_COLOR)) {
```

Comment:
NIT but did you see a case where there wasn't a color? AFAIK this should never be null, valhalla returns default values for them from gtfs specs